plugins/modules/nmcli.py

```diff
@@ -1008,6 +1008,23 @@
                 description: C(NMSettingSecretFlags) indicating how to handle the O(wireguard.private-key) property.
                 type: int
                 choices: [ 0, 1, 2 ]
+            #Attributes for WireGuard Peers
+            wgpeer_public_key:
+                description: Public key of the WireGuard peer.
+                type: str
+                required: false
```

Comment:
`required: false` is the default and should be omitted. What you need to add is `version_added` though:
```suggestion
                version_added: 10.0.0
```
Same for the others.

Reply:
updated changes